core/src/main/scala/quasar/destination/avalanche/AvalancheDestinationModule.scala

```diff
@@ -70,16 +72,20 @@ abstract class AvalancheDestinationModule[C: DecodeJson] extends DestinationModu
     def liftF[X](fa: F[X]): EitherT[Resource[F, ?], InitError, X] =
       EitherT.right(Resource.eval(fa))
 
+    lazy val sanitizedJson = sanitizeDestinationConfig(config)
+
     val init = for {
       cfg <- EitherT.fromEither[Resource[F, ?]](cfg0)
 
-      xaCfg <- EitherT.fromEither[Resource[F, ?]] {
-        transactorConfig(cfg)
-          .leftMap(errs => scalaz.NonEmptyList(errs.head, errs.tail: _*))
-          .leftMap(DE.invalidConfiguration[Json, InitError](
-            destinationType,
-            sanitizeDestinationConfig(config), _))
-      }
+      xaCfg <- EitherT(
+        Resource.eval(
+          connectionConfig(cfg).transactorConfig( 
+            auth
+          ).map(_.leftMap(s => 
+             DE.InvalidConfiguration(destinationType, sanitizedJson, scalaz.NonEmptyList(s))
+           ))
+        )
+      )
```

Comment:
Same here `)`s stay on the same line, so would all go after `scalaz.NonEmptyList(s))` here.

If you like an end marker on a new line, then we use `}` (see old code here).

The `connectionConfig(..)` expr would be formatted something like:
```
connectionConfig(cfg)
  .transactorConfig(auth)
  .map(_.leftMap(s => 
    DE.InvalidConfiguration(destinationType, sanitizedJson, scalaz.NonEmptyList(s))))
```